arch/arm64/boot/dts/qcom/sm6350-sony-xperia-lena-pdx213.dts

```diff
@@ -458,8 +768,79 @@
 		drive-strength = <2>;
 		bias-pull-down;
 	};
+
+	qup_uart1_sleep_cts: qup-uart1-sleep-cts-state {
+		pins = "gpio61";
+		function = "gpio";
+		/*
+		* Configure a bias-bus-hold on CTS to lower power
+		* usage when Bluetooth is turned off. Bus hold will
+		* maintain a low power state regardless of whether
+		* the Bluetooth module drives the pin in either
+		* direction or leaves the pin fully unpowered.
+		*/
+		bias-bus-hold;
+	};
+
+	qup_uart1_sleep_rts: qup-uart1-sleep-rts-state {
+		pins = "gpio62";
+		function = "gpio";
+		/*
+		* Configure pull-down on RTS. As RTS is active low
+		* signal, pull it low to indicate the BT SoC that it
+		* can wakeup the system anytime from suspend state by
+		* pulling RX low (by sending wakeup bytes).
+		*/
+		bias-pull-down;
+	};
+
+	qup_uart1_sleep_rx: qup-uart1-sleep-rx-state {
+		pins = "gpio64";
+		function = "gpio";
+		/*
+		* Configure a pull-up on RX. This is needed to avoid
+		* garbage data when the TX pin of the Bluetooth module
+		* is floating which may cause spurious wakeups.
+		*/
+		bias-pull-up;
+	};
+
+	qup_uart1_sleep_tx: qup-uart1-sleep-tx-state {
+		pins = "gpio63";
+		function = "gpio";
+		/*
+		* Configure pull-up on TX when it isn't actively driven
+		* to prevent BT SoC from receiving garbage during sleep.
+		*/
+		bias-pull-up;
+	};
+};
+
+&uart1 {
+	/delete-property/ interrupts;
+	interrupts-extended = <&intc GIC_SPI 602 IRQ_TYPE_LEVEL_HIGH>,
+			     <&tlmm 64 IRQ_TYPE_EDGE_FALLING>;
+
+	pinctrl-names = "default", "sleep";
+	pinctrl-1 = <&qup_uart1_sleep_cts>, <&qup_uart1_sleep_rts>, <&qup_uart1_sleep_tx>, <&qup_uart1_sleep_rx>;
+
+	status = "okay";
+
+	bluetooth {
+		compatible = "qcom,wcn3991-bt"; // Downstream DT mentions a WCN3990, but this seems incorrect
```

Comment:
according to my info SM7225 can be paired with either WCN3991 or WCN3988. Not sure about SM6350 but it's probably the same as well given it's essentially just two speedbins of the same SoC.